creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -150,14 +66,119 @@ const SecondarySyncHealthTracker = {
    * Builds redis key pattern given params
    * Key pattern string can map to one or multiple keys
    */
-  _getRedisKeyPattern (secondary, wallet, syncType, outcome, date = null) {
+  _getRedisKeyPattern ({ secondary = '*', wallet = '*', syncType = '*', outcome = '*', date = null }) {
     const prefix = 'SecondarySyncRequestOutcomes-Daily'
 
     // format: YYYY-MM-DD
     date = date || new Date().toISOString().split('T')[0]
 
     return `${prefix}:::${secondary}:::${wallet}:::${syncType}:::${date}:::${outcome}`
+  },
+
+  _parseSecondaryFromRedisKey (key) {
```

Comment:
this could also be generalized to 


```js
_parseRedisKeyPattern (key) {
  const values = key.split(':::')
  const [prefix, secondary, wallet, syncType, date, outcome] = values
  return {secondary, wallet, syncType, date, outcome}
}
```

Reply:
lol wow i love this, done